buzzmobile/sense/inputer/inputer.lisp

```diff
@@ -0,0 +1,7 @@
+(in-package :buzzmobile)
+
+(defun inputer ()
+    (with-ros-node ("inputer")
+        (let ((i 0) (pub (advertise "destination" "std_msgs/String")))
+            (loop-at-most-every .1
+                (publish-msg pub :data (read))))))
```

Comment:
Please factor out the publishing logic into its own module that can be imported everywhere